src/CoverageTransformer.js

```diff
@@ -0,0 +1,246 @@
+const { Collector } = require('istanbul');
```

Comment:
I would like to use ES6 modules but, from the practical point, at the moment, commonjs are more suitable for this particular case because we can run tests on current nodejs version without traspiling source code… So at least for now I would prefer staying with commonjs